src/components/input.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import "./input.css";
+
+export default function AskQuestion(props) {
```

Comment:
Your filename and component name should be the same.

`input.js` -> `QuestionInput.js`
`AskQuestion` -> `QuestionInput`
`input.css` -> `questionInput.css`